packages/lit-dev-content/site/tutorials/content/working-with-lists/00.md

```diff
@@ -0,0 +1,33 @@
+This tutorial will be exploring different ways of rendering a list of
+items in Lit!
+
+## Previous knowledge
+It's recommended to have an understanding of
+[defining a component](/docs/components/defining/),
+[rendering](/docs/components/rendering/),and
+[reactive properties](/docs/components/properties/) before
+working on this tutorial.<br>
+Recommended prerequisite: [Intro to Lit](/tutorials/intro-to-lit/).
```

Comment:
Nit: Found it odd (both language and format) to have a sentence above recommending certain docs, and then a non-sentence immediately below it recommending more. Maybe format as a bulleted list, or else just add the tutorial to the sentence?

Reply:
It might also be nice to have a  really brief outline here to setup what's going to happen.

Reply:
Combined the "Previous knowledge" section into sentences, and added a bulleted list of "What you'll learn"